less/_defaults/_colors.less

```diff
@@ -0,0 +1,167 @@
+// --------------------------------------------------
+// Colour definitions
+// --------------------------------------------------
+@blue: #36CDE8;
+@blueDark: #263944;
+@grey: #9096A0;
+@greyDark: #4D4D4D;
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Body, link, and heading colour
```

Comment:
I think it could be useful to include some inverted colour variables here, particularly for Authoring tool theming. 
For example, when applying background images/colours you might need to invert the text colours depending on the darkness/lightness of the background. Please see below for example.

```
@font-color: @greyDark;
@font-color-inverted: @white;

@link: @greyDark;
@link-inverted: @white;

@link-hover: @link;
@link-inverted-hover: @link;

@heading-color: @blue;
@heading-color-inverted: @white;
```